core/frontend/src/tools/ViewTool.ts

```diff
@@ -3298,8 +3298,10 @@ export class ViewGlobeLocationTool extends ViewTool {
   public static get minArgs() { return 1; }
   public static get maxArgs() { return undefined; }
 
-  // arguments: latitude longitude | string
-  // the latitude and longitude arguments are specified in degrees
+  /** This runs the tool based on the provided location arguments.
+   * arguments: latitude longitude | string
+   * the latitude and longitude arguments are specified in degrees
+   **/
```

Comment:
Elaborate on "location" and/or provide examples ("Syndney Opera House", "1600 Pennsylvania Drive, Washington DC", etc).

Reply:
good idea - thanks

Reply:
done